src/PowerShellEditorServices.Host/DebugAdapter.cs

```diff
@@ -5,124 +5,84 @@
 
 using Microsoft.PowerShell.EditorServices.Protocol.DebugAdapter;
 using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol;
+using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol.Channel;
+using Microsoft.PowerShell.EditorServices.Protocol.Server;
 using Microsoft.PowerShell.EditorServices.Utility;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
 using System.Threading.Tasks;
 
 namespace Microsoft.PowerShell.EditorServices.Host
 {
-    internal class DebugAdapter : IMessageProcessor
+    internal class DebugAdapter : DebugAdapterBase
```

Comment:
Same comment as with the LanguageServer class. But put me in my place if I'm not understanding the intent. :)